lib/facter/facter_dot_d.rb

```diff
@@ -49,7 +49,7 @@ def txt_parser(file)
       end
     end
   rescue StandardError => e
-    Facter.warn("Failed to handle #{file} as text facts: #{e.class}: #{e}")
+    Facter.warn(_("Failed to handle #{file} as text facts: #{e.class}: #{e}"))
```

Comment:
i've been told interpolated strings are a non-issue which is why i thought i could get away with it. i'll be looking into this today.